src/runtime_src/core/tools/common/XBUtilities.h

```diff
@@ -152,6 +152,9 @@ namespace XBUtilities {
   uint64_t 
   string_to_bytes(std::string str);
 
+  bool
+  isPowerOf2(const uint64_t x);

```

Comment:
I can't help notice some inconsistency in the naming convention used.
I know that tools (xbutil/xbmgmt) tend to use camelCase  whereas core XRT is lower_case_under_score style.  Here though, I see under utility functions being lower_case ones, so maybe keep is_power_of_2()?

Reply:
Fixed!